src/components/chart/_macro.njk

```diff
@@ -75,7 +75,9 @@
             {% if params.chartType in supportedChartTypes %}
                 <div data-highcharts-chart></div>
             {% else %}
-                <p data-invalid-chart-type><b>Chart type "{{ params.chartType }}" is not supported</b></p>
+                <p data-invalid-chart-type data-unsupported-chart-text>
+                    <b>{{ '"' + params.chartType + '" ' + (params.unsupportedChartText or 'chart type is not supported') }}</b>
```

Comment:
Would it make sense to have a hyphen here instead of just a space? Im not a content designer but that makes sense to me